[Doc] Proofreading documentation (#6998)
vllm-project/vllm: docs/source/performance/optimization.md

- Decrease `max_num_seqs` or `max_num_batched_tokens`. This can reduce the number of concurrent requests in a batch, thereby requiring less KV cache space.
- Increase `tensor_parallel_size`. This approach shards model weights, so each GPU has more memory available for KV cache.

You can also monitor the number of preemption requests through Prometheus metrics exposed by the vLLM. Additionally, you can log the cumulative number of preemption requests by setting ``disable_log_stats=False``.

(chunked-prefill)=

## Chunked Prefill

> vLLM supports an experimental feature - chunked prefill. Chunked prefill enables chunking large prefills into smaller chunks and batching them together with decode requests.

Reviewer: ```suggestion
vLLM supports an experimental feature: chunked prefill. Chunked prefill enables large prefills to be chunked into smaller chunks and batched together with decode requests.
```

# llm = LLM(model="meta-llama/Llama-2-7b-hf", enable_chunked_prefill=True, max_num_batched_tokens=2048)
```

> By default, vLLM scheduler prioritizes prefills and does not batch prefill, and decodes to the same batch.

Reviewer: ```suggestion
By default, vLLM scheduler prioritizes prefills and does not batch prefill and decodes in the same batch.
```
